x-pack/plugins/profiling/common/__fixtures__/stacktraces.ts

```diff
@@ -79,9 +79,9 @@ export const stackTraces = new Map([
   [
     stackTraceID.A,
     {
-      FileIDs: [fileID.A, fileID.B, fileID.C, fileID.D],
-      AddressOrLines: [addressOrLine.A, addressOrLine.B, addressOrLine.C, addressOrLine.D],
-      FrameIDs: [frameID.A, frameID.B, frameID.C, frameID.D],
+      FileIDs: [fileID.D, fileID.B, fileID.C, fileID.A],
+      AddressOrLines: [addressOrLine.D, addressOrLine.B, addressOrLine.C, addressOrLine.A],
+      FrameIDs: [frameID.D, frameID.B, frameID.C, frameID.A],
```

Comment:
The first and last were switched, but the middle elements were not. Is this as expected?

Reply:
The order of the middle frames is not relevant for the test, so why change it ?

Reply:
Ok, I changed it to not confuse readers 😃